semantic/src/site/elements/container.overrides

```diff
@@ -1,3 +1,9 @@
 /*******************************
          Site Overrides
 *******************************/
+
+@media only screen and (min-width: 1200px) {
+  .ui.container {
+    width: 1024px;
+  }
+}
```

Comment:
Why do we need this? Semantic-UI already has good default width settings for Container, I think.
Or, if the default is too wide (I think it's actually 1027px), then just over-ride the variable that has this value.

Reply:
Hmmm Ok, I can do that, but have also defined something extra for mobile - although we aren't  doing mobile for now, the container was showing all the content of application very narrow. So I added this: